### ISSUE TITLE
Systemd service file for CentOS installation is incorrect

### ISSUE THREAD

**Issue by TuurDutoit** [home-assistant/home-assistant.github.io#5656]
The systemd service file template included in the CentOS installation instructions is incorrect, I think.

This is what it looks like now:
```
[Unit]
Description=Home Assistant
After=network.target

[Service]
Type=simple
User=homeassistant
# Make sure the virtualenv Python binary is used
Environment=VIRTUAL_ENV="/srv/homeassistant"
Environment=PATH="$VIRTUAL_ENV/bin:$PATH"
# ExecStart using software collection:
ExecStart=/usr/bin/scl enable rh-python36 -- /srv/homeassistant/bin/hass -c "/home/homeassistant/.homeassistant"

[Install]
WantedBy=multi-user.target
```

The problem is that systemd won't do any variable substitution in this file, as mentioned in the manual pages ([systemd.exec(5)](https://www.freedesktop.org/software/systemd/man/systemd.exec.html#Environment)). This means `$VIRTUAL_ENV` and `$PATH` will be left as-is in the `PATH` variable, resulting in the final value `"$VIRTUAL_ENV/bin:$PATH"`. This obviously won't work.

I hit this issue as HomeAssistant couldn't find the `ping` command. Substituting the HomeAssistant entry file with a simple test file, I confirmed the `PATH` was equal to `/opt/rh/rh-python36/root/usr/bin:"$VIRTUAL_ENV/bin:$PATH"` (I assume scl adds `/opt/rh/rh-python36/root/usr/bin` to the `PATH`). The issue was resolved by commenting out the `Environment=` lines.

Is this template indeed incorrect, or am I doing something wrong here?

**Comment by brgerig** [comment]
I just found this issue after spending several hours figuring out everything you documented here. This issue really needs to be re-opened and addressed.

I actually created a Home Assistant script to put the output of `echo $PATH` into a file in `/tmp`, then manually substituted that into the Environment line.

In addition, the quotes are in the wrong place anyway. Here's what my applicable line of that file looks like:
`Environment="PATH=/opt/homeassistant/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin"`

(I use `/opt`, not `/srv`.)
